Classes/BITCrashCXXExceptionHandler.mm

```diff
@@ -58,7 +58,7 @@
 
 @implementation BITCrashUncaughtCXXExceptionHandlerManager
 
-extern "C" void LIBCXXABI_NORETURN __cxa_throw(void *exception_object, std::type_info *tinfo, void (*dest)(void *))
```

Comment:
Might not build against the old headers then. Could use __attribute__ ... directly though.